### PR TITLE
Increment to release `v1.2.2-pre0`

### PR DESCRIPTION
**Pull request by the-real-adammork**
Soundtrack of this PR: [Theo Parrish on NTS](https://www.mixcloud.com/NTSRadio/theo-parrish-120622/)

### Motivation

Increment for a release

### In this PR
* SwiftNIO package dependency update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.1-pre0] - 2022-06-24
```
Comment (collinjackson) [inline CHANGELOG.md]: Pre-release versions have a lower precedence than the associated normal version. Are you sure you don't want to make this 1.2.2-pre0?

Comment (the-real-adammork) [reply CHANGELOG.md]: right thats a good point, Ive updated the version to be `1.2.2-pre0`

**Review by voloshyn — APPROVED** [review]
LGTM

**Review by collinjackson — APPROVED** [review]
LGTM

**Pull request merged**